.github/workflows/node.js.yml

```diff
@@ -9,29 +9,27 @@ on:
 
```

Comment:
This is one line off because I can't comment on the actual line. 

Do we still to be using `pull_request_target`? Is there a benefit to that?